eZ/Publish/API/Repository/Values/User/Role.php

```diff
@@ -23,7 +23,24 @@
 abstract class Role extends ValueObject
 {
     /**
-     * ID of the user rule.
+     * @var int Status constant for defined (aka "published") role
+     */
+    const STATUS_DEFINED = 0;
+
+    /**
+     * @var int Status constant for draft (aka "temporary") role
+     */
+    const STATUS_DRAFT = 1;
+
+    /**
+     * @var int Status constant for modified (aka "deferred for publishing") role
+     *
+     * Reserved for future use, e.g. to delay cache invalidation after a role change.
+     */
+    const STATUS_MODIFIED = 2;
```

Comment:
@lolautruche I discussed with AR to use it for delayed/async role cache clearing. But we can of course remove it until it's needed.